package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "pwa-kit",
-  "version": "3.0.0-preview.0",
+  "version": "3.0.0-preview.3",
```

Comment:
Since this PR merged in the latest changes from the develop branch, you can filter for just the package.json files to verify the version changes only.